### PR TITLE
feature: add api platform stats and console achievements

### PR DESCRIPTION
**Pull request by Simperfit**
Adding the achievements on the console components, adding the stats for API-Platform.

### CODE REVIEW

File: achievements/console.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Symfony Console
+
+In Symfony Console we've fixed some bugs that were blocking some contributor to run the tests and to contribute even more to the component.
```
Comment (greg0ire) [inline achievements/console.md]:
```suggestion
In the Symfony Console component, we've fixed some bugs that were preventing
some contributors to run the tests independently from each other and to
contribute even more to the component.
```

File: achievements/console.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+We started talking about how we could refactor the SymfonyStyle and blocks in order to have something more usable and maintenable. The fact that core team members and contributors meet helped alot to fix some of the bugs.
```
Comment (greg0ire) [inline achievements/console.md]:
```suggestion
We started talking about how we could refactor the SymfonyStyle and blocks in
order to have something more usable and maintenable. The fact that core team
members and contributors meet helped a lot to fix some of the bugs.
```

File: achievements.md [hunk 3]
```diff
@@ -1,6 +0,0 @@
-# Achievements
-
-During the hackathon, attendees worked on different activities, not only coding but also reflecting on different topics such as diversity within the community.
-
-## Repositories statistics
-
```
Comment (greg0ire) [inline achievements.md:6]: Why is this file removed?

Comment (Simperfit) [reply achievements.md:6]: seems not to be used anywhere

**Review by jfhovinne — APPROVED** [review]
Looks good, thanks!
